### PR TITLE
CI: Revive Banklist Tests

### LINKED ISSUE #38988: BUG: Banklist.html was removed, failure on master
**Note**: Please read [this guide](https://matthewrocklin.com/blog/work/2018/02/28/minimal-bug-reports) detailing how to provide the necessary information for us to reproduce your bug.

#### Code Sample, a copy-pastable example

The banklist.html file was removed from "https://www.fdic.gov/resources/resolutions/bank-failures/failed-bank-list/
#38987 marked the tests as xfail and removed the userguide entry.

#### Expected

Replace the xfail tests with a new html file url and adjust the userguide appropriately.

### PR DESCRIPTION
**Pull request by lithomas1**
- [x] closes #38988
- [x] tests added / passed
- [x] Ensure all linting tests pass, see [here](https://pandas.pydata.org/pandas-docs/dev/development/contributing.html#code-standards) for how to run them

### CODE REVIEW

File: pandas/tests/io/test_html.py [hunk 1]
```diff
@@ -134,28 +134,26 @@ def test_to_html_compat(self):
         res = self.read_html(out, attrs={"class": "dataframe"}, index_col=0)[0]
         tm.assert_frame_equal(res, df)
 
-    @pytest.mark.xfail(reason="Html file was removed")
     @tm.network
     def test_banklist_url_positional_match(self):
-        url = "https://www.fdic.gov/bank/individual/failed/banklist.html"
```
Comment (alimcmaster1) [inline pandas/tests/io/test_html.py:140]: Think this just redirects to the same as the below?

**Review by alimcmaster1 — CHANGES_REQUESTED** [review]
Can you also fix up PEP8 and see if the test failures are fixable?

@lithomas1 - happy to help review this one

**Comment by jreback** [comment]
can you rebase

**Comment by lithomas1** [comment]
This is un-mergeable in its current state because the lxml engine is inconsistent with the other engines which do read this in properly. It has to do with the ``attrs`` keyword, which when unspecified makes lxml return the correct result.

I do not have much time to patch the lxml engine right now because school has started again, and I am mostly focusing my efforts on other PRs.

Because the doc update just does a plain ``read_html``, I can update the PR to do a doc update only, if you want.

**Comment by jreback** [comment]
i see
we can just remove those 2
tests entirely as well

**Comment by lithomas1** [comment]
@github-actions pre-commit.

File: pandas/tests/io/test_html.py (hunk 1 above)
Comment (lithomas1) [reply pandas/tests/io/test_html.py:140]: Yeah, we shouldn't rely on them to keep redirecting it for us, so I changed it.

**Comment by lithomas1** [comment]
I removed the attrs checking component, and commented it. An incomplete test is better than no test I guess...

File: pandas/tests/io/test_html.py (hunk 1 above)
Comment (alimcmaster1) [reply pandas/tests/io/test_html.py:140]: thanks!

**Comment by alimcmaster1** [comment]
thanks @lithomas1 - LGTM

**Pull request merged**